src/pages/admin/apply/[id]/index.tsx

```
@@ -0,0 +1,225 @@
+    if (typeof window !== 'undefined') {
+      return sessionStorage.getItem(getStorageKey(id)) || 0;
+    }
+    return 0;
+  });
+
+  const handleTabChange = (index: number) => {
+    setActiveTab(index);
+    sessionStorage.setItem(getStorageKey(id), String(index));
+  };
+
+  const { data: data, isLoading } = useAllApplication(Number(id), token);
```

Biome (1.9.4)

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)